getting-started/logging-in.rst

```diff
@@ -45,7 +45,14 @@ Step 2: Log in
 ==============
 
 Open `lsst-lsp-stable.ncsa.illinois.edu/nb <https://lsst-lsp-stable.ncsa.illinois.edu/nb>`__ in your web browser.
-Click the **Sign in with CILogon** button, then enter your NCSA Kerberos credentials on the NCSA CILogon page.
+Click the **Sign in with CILogon** button.
+This will take you to a page where you can choose an identity provider.
+You must choose NCSA as the provider the first time you log in (type NCSA in the search box to find it).
+Once you have chosen NCSA as the identity provider, click the **Log On** button and you will be taken to a page where you can enter your NCSA Kerberos credentials.
+
+You can add identity providers using the identity management page at https://identity.lsst.org/manage.
```

Comment:
This doesn't belong in "Step 2"; it's a completely separate procedure.

I think what I'd do is add a new section called "Link your login with another identity provider" Actually it probably should be a separate page: `getting-started/identity-providers.rst`. That page should explain what identity providers are, why you'd want to do that, and the steps: VPN (?), go to the identity page, add identity, ...

Then link to that section from this step, and also from the "Next steps" section.